### ISSUE TITLE
TestImagePruneErrOnBadReference flake

### ISSUE THREAD

**Issue by jim-minter** [openshift/origin#17223]
https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/17215/test_pull_request_origin_unit/5323/

```
github.com/openshift/origin/pkg/oc/admin/prune TestImagePruneErrOnBadReference 0.02s

images_test.go:142: pruner error: failed to build graph - no changes made
images_test.go:107: pruner error output: Failed to build graph!

The following objects have invalid references:

Pod[foo/pod1]: invalid docker image reference "invalid image reference": invalid reference format
BuildConfig[foo/bc1]: invalid ImageStreamImage reference "bar:invalid-digest": expected exactly one @ in the isimage name "bar:invalid-digest"
Deployment[foo/dep1]: invalid docker image reference "do not blame me": invalid reference format

Either fix the references or delete the objects to make the pruner proceed.

images_test.go:145: bump master version and try again
images_test.go:153: pruner error: failed to build graph - no changes made
images_test.go:107: pruner error output: Failed to build graph!
Failed to get master API version: error: server took too long to respond with version information.

The following objects have invalid references:

Pod[foo/pod1]: invalid docker image reference "invalid image reference": invalid reference format
BuildConfig[foo/bc1]: invalid ImageStreamImage reference "bar:invalid-digest": expected exactly one @ in the isimage name "bar:invalid-digest"
Deployment[foo/dep1]: invalid docker image reference "do not blame me": invalid reference format

Either fix the references or delete the objects to make the pruner proceed.

images_test.go:126: expected message "client version" is not contained in the output
```

**Comment by jim-minter** [comment]
**Failed to get master API version: error: server took too long to respond with version information.**

The PR in question included 58076ba560828b50c15ed250de75044b4c8283a1 - not possible that a longer timeout is required?

**Comment by miminar** [comment]
I could reproduce similar issue when shortening the timeout to 100 milliseconds. So I guess increasing to a second will be enough. Done by #17235
